docs/api.md

```diff
@@ -111,61 +111,84 @@ player.playlist.currentItem();
 // -1
 ```
 
-#### `player.playlist.addItem(String|Object newItem, [Number index])`
+#### `player.playlist.add(String|Object|Array items, [Number index])`
 
 Adds an item to the current playlist without replacing the playlist.
 
-Fires the `playlistitemadded` event.
+Fires the `playlistadded` event.
```

Comment:
We should mention that `playlistchange` is not fired in this doc too, since it could easily be assumed otherwise.

Reply:
should playlistchange fire for added/removed events too?

Reply:
also, I think this is the `playlistadd` event, not `playlistadded`
https://github.com/videojs/videojs-playlist/blob/50503ca20bb935652ac785b04f9f0d549f975401/src/playlist-maker.js#L410

Reply:
Yeah, I was going back and forth on `playlistchange` but I will restore it.